portfolio.html

```diff
@@ -20,46 +20,279 @@
         </nav>
       </header>
       <main>
-        <section class="headline">
-          <h2 class="title">I’m Yaseen Glad to see you!</h2>
+        <section class="headline bg-white">
+          <h1 class="title">I’m Yaseen Glad to see you!</h1>
           <p class="details">
             I’m a software developer! I can help you build a product , feature
             or website Look through some of my work and experience! If you like
             what you see and have a project you need coded, don’t hestiate to
             contact me.
           </p>
-          <p></p>
-          <h2 class="contact-title">LET’S CONNECT</h2>
-          <ul class="social-media">
-            <li>
-              <a href="#"
-                ><img src="images/social-media-icons/tw.png" alt="Twitter"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img
-                  src="images/social-media-icons/linkedin.png"
-                  alt="LinkedIn"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img src="images/social-media-icons/medium.png" alt="Medium"
-              /></a>
+          <div class="social-media">
+            <a class="contact-link" href="#">LET’S CONNECT</a>
+            <ul class="social-media-links">
+              <li>
+                <a href="#"
+                  ><img src="images/social-media-icons/tw.png" alt="Twitter"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img
+                    src="images/social-media-icons/linkedin.png"
+                    alt="LinkedIn"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="images/social-media-icons/medium.png" alt="Medium"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="images/social-media-icons/github.png" alt="Github"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="images/social-media-icons/v.png" alt="V"
+                /></a>
+              </li>
+            </ul>
+          </div>
+        </section>
+        <section class="works">
+          <article class="card bg-white">
+            <figure>
+              <img src="images/portfolio/tonic.png" alt="Tonic" />
+            </figure>
+            <div class="project-info">
+              <div class="project-header">
+                <h1 class="project-title">Tonic</h1>
```

Comment:
- [ ] Kindly respect the order of headings in the HTML document to improve your page's accessibility. so you should have an `h1` tag in your document but other headings are allowed to be used multiple times. you got this 👍